test/unit/fix-request-body.spec.ts

```diff
@@ -52,6 +52,40 @@ describe('fixRequestBody', () => {
     expect(proxyRequest.write).toHaveBeenCalledWith(expectedBody);
   });
 
+  it('should write when body is not empty and Content-Type is multipart/form-data', () => {
+    const proxyRequest = fakeProxyRequest();
+    proxyRequest.setHeader('content-type', 'multipart/form-data');
+
+    jest.spyOn(proxyRequest, 'setHeader');
+    jest.spyOn(proxyRequest, 'write');
+
+    const body = new FormData();
+    body.append('someField', 'some value');
+
+    fixRequestBody(proxyRequest, { body } as Request<express.Request>);
+
+    const expectedBody = handlerFormDataBodyData('multipart/form-data', body);
+    expect(proxyRequest.setHeader).toHaveBeenCalledWith('Content-Length', expectedBody.length);
+    expect(proxyRequest.write).toHaveBeenCalledWith(expectedBody);
+  });
+
+  it('should write when body is not empty and Content-Type includes multipart/form-data', () => {
+    const proxyRequest = fakeProxyRequest();
+    proxyRequest.setHeader('content-type', 'multipart/form-data');
+
+    jest.spyOn(proxyRequest, 'setHeader');
+    jest.spyOn(proxyRequest, 'write');
+
+    const body = new FormData();
+    body.append('someField', 'some value');
+
+    fixRequestBody(proxyRequest, { body } as Request<express.Request>);
+
+    const expectedBody = handlerFormDataBodyData('multipart/form-data', body);
```

Comment:
Same. Please remove the call to `handlerFormDataBodyData` to generate `expectedBody`